corehq/apps/smsforms/management/commands/migrate_sms_sessions_to_sql.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management.base import BaseCommand
+from corehq.apps.smsforms.models import XFormsSession, sync_sql_session_from_couch_session, SQLXFormsSession
+from dimagi.utils.couch.database import iter_docs
+
+
+class Command(BaseCommand):
+    args = ""
+    help = ""
+
+    def handle(self, *args, **options):
+        db = XFormsSession.get_db()
+        session_ids = [row['id'] for row in db.view("smsforms/sessions_by_touchforms_id")]
```

Comment:
I think it will be slow but return. it's just returning ids for each doc and we do something similar in all the `ptop_fast_reindex` commands for things that have more docs like forms and cases.